### PR TITLE
Update Win Pokemon Reducer

### LINKED ISSUE #133: Win Pokemon Reducer
**Is your feature request related to a problem? Please describe.**
Need a reducer to handle winning a pokemon

**Describe the solution you'd like**
Write a reducer that subtracts a set value on each spin

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. When a user plays the game, the value it cost to play the game is removed from their total money. 

## Purpose
This will make it so the user cannot just infinitely play the game.

## Approach
Adjust the pokeDollar state in the winPokemon reducer. 

Closes #133

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks nice! Keep up the great work.

File: pokeapp/src/redux/pokemonSlice.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const pokemonSlice = createSlice({
   //initialize states
   initialState: {
     allPokemon: [],
-    pokeDollars: 1000000,
+    pokeDollars: 10000,
```
Comment (Lap343) [inline pokeapp/src/redux/pokemonSlice.js:10]: $$$

File: pokeapp/src/components/GamePage.js [hunk 2]
```diff
@@ -147,6 +148,19 @@ const GamePage = ({
     }
     return currStyle;
   };
+
+  const playGame = () => {
+    if(pokeDollars - 1000 > 0){
```
Comment (Lap343) [inline pokeapp/src/components/GamePage.js:153]: So I need at least 1000 pokedollars to play. Interesting.

**Review by Lap343 — APPROVED** [review]
Very good code. Also like the setTimeouts so the game will look like actual slots

File: pokeapp/src/redux/pokemonSlice.js (hunk 1 above)
Comment (JorgeLVilla) [inline pokeapp/src/redux/pokemonSlice.js:10]: Gimme the money!

**Review by JorgeLVilla — APPROVED** [review]
Nice job! Excited to see the game.

**Pull request merged**